doc/releases/release-notes-2.1.rst

```diff
@@ -96,6 +96,11 @@ Build and Infrastructure
   Use new functions that utilize eDTS info such as dt_node_reg_addr.
   See :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for details.
 
+* Deprecated direct use of the `DT_` Kconfig symbols from the generated
+  `generated_dts_board.conf`.  We should now utilize either functions from
+  `scripts/kconfig/kconfigfunctions.py` for kconfig or functions in
+  `scripts/sanity_chk/expr_parser.py` for sanitycheck.
```

Comment:
Needs two backticks or `` :file:`scripts/sanity_chk/expr_parser.py` ``.

Think `` :zephyr_file:`scripts/sanity_chk/expr_parser.py` `` will turn it into a link, so that might be best.